plugins/collapsible-layers.js

```diff
@@ -14,26 +14,33 @@ collapsibleLayers.initial = true; // collapsed
 function setup () {
   var CollapsibleLayers = L.LayerGroup.extend({
     initialize: function (match, options) {
-      this._ctrl = window.layerChooser;
-
-      L.Util.setOptions(this, options);
+      options = options || {};
       options.notPersistent = true;
+      L.Util.setOptions(this, options);
+
       this._name = options.name || match;
+      this._ctrl = window.layerChooser;
       this._layers = {};
```

Comment:
`this._layers` is never used

Reply:
It is used in parent class.
There is also `clearLayers` method but it has undesirable (for our case) side effects

Reply:
then, calling `LayerGroup.initialize.call(this.[],options)` sounds more idiomatic

Reply:
Still heavily dependent on parent implementation, so in the end there is no difference.